run_tests.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -e
+swift run &
+PID=$!
+kill -INT $!
```

Comment:
Why did you make this change? I don't understand the second sentence of your commit message.

Reply:
This was changed because the only way `data.txt` can be built is if the server runs. In this case I run the server and kill it so when `swift test` runs a `data.txt` file now exists and the tests will run without an error.

Reply:
Hmm. Starting and stopping the server is pretty heavy-handed for that. What other ways have you considered to ensure that the file exists?

Reply:
Agreed with Craig on this one.